script.js

```diff
@@ -22,3 +22,223 @@ navLinksEl.forEach((item) => {
+    FeaturedImage: 'images/Snapshoot Portfolio.png',
+  },
+
+  {
+    Name: 'Uber Navigation',
+    Description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    FeaturedImage: 'images/Snapshoot Portfolio (3).png',
+  },
+];
+
+const sectionHide = document.querySelector('.container');
+const btnEl1 = document.querySelector('.btn-one');
+const btnEl2 = document.querySelector('.btn-two');
+const btnEl3 = document.querySelector('.btn-three');
+const btnEl4 = document.querySelector('.btn-four');
+const body = document.querySelector('body');
+
+btnEl1.addEventListener('click', () => {
+  sectionHide.classList.toggle('hide-container');
+  body.classList.toggle('background-img');
+  const Popsection = document.createElement('section');
+  Popsection.className = 'Popup-Section';
+  Popsection.innerHTML = `
+      <div class="icon-holder">
+      <h2>${workCards[0].Name} </h2>
+      <a onclick="showMenu(close)"><i class="fa-solid fa-xmark close-cards"></i></a>
+      </div>
+      <article class="wrap-frame">
+      <p class="frame canopy">CANOPY</p>
+      <img src="images/Counter.png" alt="">
+      <p class="frame">Back End Dev</p>
+      <img class="frame" src="images/Counter.png" alt="" class="card-img">
+      <p class="frame">2015</p>
+      </article>
+      <img src= "${workCards[0].FeaturedImage}" alt="Snapshoot" class="card-img desk-img"> 
+      <div class="right-pop">
```

Comment:
- [ ] Good job 😃 at implementing the popup functionality, but I see the position style of the popup needs to go right little to center it on the page to be like the Figma Design💯
- [ ] Also consider the image of the popup needs some repairs on the desktop version I see that the size of the image of the popup is bigger than the Figma Design💯, so kindly try to make the size of the image like the Figma Design to be suitable for the eye and with that the portfolio will be attractive for the clients who will communicate with you from your portfolio😊

> Check below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|    ![image](https://user-images.githubusercontent.com/83360636/216096467-d51e7715-2617-40aa-8909-c50b918bb4ae.png)   | ![image](https://user-images.githubusercontent.com/83360636/216096631-ee13513a-9aa1-4642-9318-73ed33e73d75.png)  |